Cargo.toml

```diff
@@ -1,3 +1,4 @@
+cargo-features = ["edition"]
```

Comment:
This isn't required as of Rust 1.31, which is going to be a requirement for compiling the next release, anyway. Can we please remove this?